### ISSUE TITLE
Is the Refs footer supposed to be missing a :?

### ISSUE THREAD

**Issue by daotoad** [conventional-commits/conventionalcommits.org#399]
The example in the main index.md showing a commit with a Refs footer (https://github.com/conventional-commits/conventionalcommits.org/blob/master/content/v1.0.0/index.md#commit-message-with-multi-paragraph-body-and-multiple-footers) shows no ':' delimiter.

```
fix: correct minor typos in code

see the issue for details

on typos fixed.

Reviewed-by: Z
Refs #133
```

Is that correct?  Are there particular footers that are special?  Or is this just a typo?

**Comment by damianopetrungaro** [comment]
Fixed @daotoad  😄 !

**Comment by oncet** [comment]
Sorry to bring this up but the specifications states:

> One or more footers MAY be provided one blank line after the body. Each footer MUST consist of a word token, followed by either a :<space> or <space># separator, followed by a string value

My interpretation is that a footer should either use one or the other, not both at the same time.

Good:
```
Refs #133
```

Bad:
```
Refs: #133
```

**Comment by LemmingAvalanche** [comment]
The `:<space>` / `<space>#` dichotomy comes from the [effort](https://github.com/conventional-commits/conventionalcommits.org/issues/179#issue-481856441) to be optionally compatible with [git trailers](https://git-scm.com/docs/git-interpret-trailers). And git-interpret-trailers(1) made the separator flexible (you can use `%<space>` for that matter) in order to be compatible with the already established at the time `Fixes #98` GitHubism.

I don’t know how this tool handles it in general but I only use `:<space>` in order to be compatible with the built-in git(1) tool.
